src/portal/views/Twin.vue

```diff
@@ -44,12 +44,10 @@
         <h2>Twin Details</h2>
       </v-card>
       <v-card class="my-3 pa-3 text-center">
-        <v-list>
-          <v-list-item> ID: {{ twin.id }} </v-list-item>
-
-          <v-list-item> Address: {{ twin.address }} </v-list-item>
-
-          <v-list-item> Relay: {{ twin.relay }} </v-list-item>
+        <v-list v-if="$store.state.credentials.initialized">
+          <v-list-item> ID: {{ $store.state.credentials.twin.id }} </v-list-item>
+          <v-list-item> Address: {{ $store.state.credentials.accountAddress }} </v-list-item>
+          <v-list-item> Relay: {{ $store.state.credentials.twin.relay }} </v-list-item>
```

Comment:
the relay should be decoded.

Reply:
we decode it in the `setCredentials` method